src/Directory.Build.targets

```diff
@@ -76,7 +71,6 @@
       <NuspecProperties>$(NuspecProperties);Copyright=$(Copyright);Description=$(Description);Title=$(Title)</NuspecProperties>
       <NuspecProperties>$(NuspecProperties);RepositoryType=$(RepositoryType);RepositoryCommit=$(RepositoryCommit);RepositoryUrl=$(RepositoryUrl)</NuspecProperties>
       <NuspecProperties>$(NuspecProperties);ProjectUrl=$(ProjectUrl);ProjectFolder=$(MSBuildProjectDirectory)</NuspecProperties>
-      <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
The snupkg was causing that Nuget warning, so we can stop ignoring that warning now.

Reply:
> that Nuget warning

Could you be more specific?

Reply:
I'm guessing you meant this in the end:

```
    <!-- https://github.com/NuGet/Home/issues/10665 -->
    <NoWarn>$(NoWarn);NU5128</NoWarn>
```